ruby_enumerable_methods.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    for item in self do
+      yield(item)
+    end
+  end
```

Comment:
- enumerables my_each should return an Enumerator if no block is given
     Failure/Error: expect(array.my_each).to be_an(Enumerator)